doc/source/installing.rst

```diff
@@ -38,22 +38,22 @@ To install using pip, simply run:
 You may also need to install some system dependencies (depending on your host).
 These are:
 
-- `OpenSSL <https://openssl.org>`_ used to generate and verify RSA signatures, and to export and verify signatures created with gpg.
-- `GPG <https://gnupg.org>`_ if you plan on generating gpg signatures (verifying
-  works without gpg).
+- `OpenSSL <https://openssl.org>`_ used to generate and verify RSA signatures, and to export and verify signatures created with GPG.
+- `GPG <https://gnupg.org>`_ if you plan on generating GPG signatures (verifying
```

Comment:
I wonder if we'd like to say `if you plan on generating PGP signatures` instead.

Reply:
Or we say `[PG]{3}` and let the reader decide, additionally we should add a reference to [RFC 4880 -- 1.1. Terms](https://tools.ietf.org/html/rfc4880#section-1.1) :)